driver-mct/main/seq_diagBGC_mct.F90

```diff
@@ -932,8 +970,16 @@ subroutine seq_diagBGC_ice_mct( ice, frac_i, infodata, do_i2x, do_x2i)
     ko    = mct_aVect_indexRA(frac_i,ofracname)
 
     if (present(do_i2x)) then
-!       if (first_time) then
-!       endif
+       if (first_time) then
```

Comment:
Is it worth a comment to explain why handling the first time differently?

Reply:
Always worth a comment! Throughout the budget codes we use "first_time" as a logic wrapper to set indices used over the course of the run that only need to be set once. Previously there were no BGC fields included in the carbon from the ice to the cpl (i2x), so the logic was commented out. This PR adds the fluxes that are exchanged between the ice and ocn through the cpl, so now it is needed. I'll add a short description of first_time to the code to make that more clear.